app/javascript/src/scenes/SettingsPage/scenes/teams/index.jsx

```diff
@@ -22,7 +22,7 @@ const Wrapper = styled.div`
 const SettingsTeams = ({ teams }) =>
   <Wrapper>
     <Breadcrumb>
-      <Breadcrumb.Item active={false}>
```

Comment:
Is OK to remove the false? Actually why is this even here if you don't want to mark the item as active you just remove the property

Reply:
It should be true, so short version of `active={true}` is just `active`. ESLint said so.